community/dbms/src/main/java/org/neo4j/commandline/dbms/DiagnosticsReportCommand.java

```diff
@@ -128,6 +129,14 @@ public void execute( String[] stringArgs ) throws IncorrectUsage, CommandFailed
         }
     }
 
+    private String getDefaultFilename() throws UnknownHostException
+    {
+        String hostName = InetAddress.getLocalHost().getHostName();
+        String safeFilename = hostName.replaceAll( "[^a-zA-Z0-9._]+", "_" );
```

Comment:
Minot thing: i think it would be good to compile this pattern independently and just use matcher. But also do not think its important here that much so will leave it up to you.